### DISCUSSION TITLE
Wyze v2 model WYZEC2

### DISCUSSION THREAD

**Discussion by languy99** [gtxaspec/wz_mini_hacks#602] [Q&A]
Hello

I have tried everything including downgrading to version 4.9.8.1002 to get SSH and I can’t. The app can see the SD card just fine. I have even tried to set up the web interface and no go. I noticed that C2 version is not listed on the compatibility chart. Does mini_havks not work on this version of the camera? FCC ID: 2ANJHWYZEC02 HVIN: WYZEC02

**Comment by shadow2354** [comment]
I also have this question-- Same model.

**Comment by virmaior** [comment]
I don't have a v2 but 

1. Regarding the SD card being visible in the app.

> the app can see the SD card

isn't super informative because once a Wyze Cam is fully booted in can use ExFAT cards. But the bootloader can only read FAT32. Are you sure you've got a FAT32 SD card in there?

2. In the case of a V2, you have to compile some tuff. Did you follow the directions at https://github.com/gtxaspec/wz_mini_hacks/wiki/Setup-&-Installation ?

**Reply by languy99** [reply]
Hello, 

Sorry for the late reply, yes I made sure it was a FAT32 card format and I followed the steps and complied all of the required files. When I do the reboot holding the setup button the camera just gets stuck in a boot loop and never boots completely.

**Comment by Semag** [comment]
@languy99  / @shadow2354  - did you guys get this working at all? I have the same issue. Compiled the demo.bin just fine after getting the proper version 4.9.8.1002. I even used the *same* SD Card to flash the camera back to the actual 4.9.8.1002 version. Then tried to flash the modified demo.bin and it gets into a boot loop.

If you remove the SD Card, it boots up just fine, it's only when trying to flash it fails.

**Reply by languy99** [reply]
No not yet. I will try again this week and see if I can get it working correctly.

**Comment by jolan** [comment]
I had 10 new-in-box WYZEC2BK (v2 Black model) that had never been set up before.  What worked for me was:

- Download/flash regular 4.9.8.1002
- Run setup via Wyze app
- copy SD_ROOT and modified demo.bin from v2install
- Flash modified firmware

Then the cameras were able to boot wz_mini OK without further interaction.

Trying to do the setup process with the modified demo.bin always failed for me and I also had boot loops with the modified firmware if the camera had never been setup.

**Comment by SalehAce1** [comment]
I have the WYZEC2 model as well and tried the steps @jolan listed with little success. I was able to flash the vanilla 4.9.8.1002 firmware and set it up just fine but when I try to flash the modded firmware it gets stuck in a loop of making a clicking sound, yellow light turning on then turning off. 

I'm not sure if my file structure is correct but I tried having the modded demo.bin as well as the SD_ROOT folder in my SDcard as well as having the contents of the SD_ROOT folder (wz_mini and factory_t31_ZMC6tilDQN) with the modded demo.bin. Neither of which worked. Before testing both of these I first reset back to the 4.9.8.1002 firmware and made sure the camera worked.

Am I doing this right/is there anything I could be missing? Any help would be greatly appreciated <3

Edit: It works on by Wyze Cam V3 so I have no idea what is going on

**Reply by Bob-AOL** [reply]
v2s are not factory_t31_ devices.

**Reply by CallSign1** [reply]
V2s are T20 based devices

**Comment by CallSign1** [comment]
I also have the WYZEC2 model and have tried the steps @jolan listed with NO success.  Everything worked perfectly on the V3s, but none of the V2s can connect to the wifi network after flashing the modded firmware.  I tried several SD cards, formatting tools, and even wifi networks with different password lengths and encryption levels.

**Comment by bjs-pdx** [comment]
I had problems with the Black V2s but did get them to work.  I think the setup program, when run, tries to download the wrong Wyze firmware for the build.  I manually put a copy of 4.9.8.1002 in the V2 build directory and that seemed to fix things.  Also had to manually activate the NIC drivers in the config.  I did have to setup the cameras in the Wyze app first, before using the wz_mini_hacks sd card for the bootup.

**Reply by hua-bing** [reply]
Got a flashing blue and yellow light after flashing the demo.bin.  Seems having issues with wifi?  Did you just change this line to "true" in wz_mini.conf?
`BONDING_ENABLED="false"`
Thanks.

**Reply by hua-bing** [reply]
Never mind.  Made the change and it's now working.  Curious why this is default to "false".  Looks to me, it should be turned on by default.

**Comment by CallSign1** [comment]
I am having some success finally.  I am able to load the modified 4.9.8.1002 firmware and have it connect to my wifi (still wide open with no encryption, for now), if I don't change anything in the wz_mini.conf file, except to enable the web server with authentication.  This has worked on two V2 cameras now.  After they boot with the modified firmware, I enable the functions I want.  Success.  I hope this helps others.
